lms/djangoapps/courseware/permissions.py

```diff
@@ -3,6 +3,7 @@
 """
 
 from bridgekeeper import perms
-from .rules import HasAccessRule
+from .rules import HasAccessRule, HasStaffAccessToContent
 
 perms['courseware.view_course_home'] = HasAccessRule('load')
+perms['courseware.masquerade_as_student'] = HasStaffAccessToContent()
```

Comment:
Maybe rebase this on top of master, and also add a constant (assuming we want to continue using constants).

Reply:
Yup, I should do that.